usaspending_api/api_contracts/contracts/v2/references/filter_tree/psc/group/group.md

```diff
@@ -0,0 +1,70 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# PSC
+
+These endpoints are used to power USAspending.gov's PSC search component on the advanced search page.
+The response is a forest of filter search nodes, which despite having a unified structure represent different
+database fields based on depth in the tree.
+
+## Toptier Search [GET /api/v2/references/filter_tree/psc/{group}/{?depth}{?filter}]
+
+Returns PSCs under the provided group
++ Request (application/json)
+    + Parameters
+        + `group`: `Product` (required, string)
```

Comment:
Does `group` only support a set list of values? If yes please turn into an enum[string]